pkg/skaffold/sync/sync.go

```diff
@@ -138,6 +142,20 @@ func inferredSyncItem(a *latest.Artifact, e filemon.Events, builds []build.Artif
 	return &Item{Image: tag, Copy: toCopy}, nil
 }
 
+func autoSyncItem(ctx context.Context, a *latest.Artifact, e filemon.Events, builds []build.Artifact) (*Item, error) {
+	tag := latestTag(a.ImageName, builds)
+	if tag == "" {
+		return nil, fmt.Errorf("could not find latest tag for image %s in builds: %v", a.ImageName, builds)
+	}
+
+	// placeholder
+	_ = ctx
+	_ = e
```

Comment:
You can make the argument names be `_` to indicate that they're unused.

Reply:
:heavy_check_mark: ah good tip. I think I'll hold off on this PR until the jib specific stuff is merged?